content/hi/cloud-computing.md

```diff
@@ -0,0 +1,26 @@
+---
+title: क्लाउड कंप्यूटिंग (Cloud Computing)
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+
+क्लाउड कंप्यूटिंग एक ऐसा मॉडल है जो इंटरनेट पर मांग पर सीपीयू, नेटवर्क और डिस्क क्षमताओं जैसे कंप्यूट संसाधनों की पेशकश करता है।
+क्लाउड कंप्यूटिंग उपयोगकर्ताओं को दूरस्थ भौतिक स्थान में कंप्यूटिंग शक्ति तक पहुँचने और उपयोग करने की क्षमता देती है।
+क्लाउड प्रदाता जैसे ऐ डब्लयू स (AWS), जी सी पी(GCP), अज़ोर(Azure), डिजिटल ओसियन (DigitalOcean), और अन्य सभी तृतीय पक्षों की पेशकश करते हैं
+कई भौगोलिक स्थानों में संसाधनों की गणना करने के लिए किराए पर लेने की क्षमता देता है।
```

Comment:
```suggestion
और अन्य सभी तृतीय पक्षों को कई भौगोलिक स्थानों में संसाधनों की गणना करने के लिए किराए पर लेने की क्षमता प्रदान करते हैं।
```

Reply:
Garima Mam can you please share with me what changes I have to make in line 11-13.

Reply:
@Garima-Negi Mam can you please share with me what changes I have to make in line 11-13.